Gemfile

```diff
@@ -0,0 +1,15 @@
+ruby '2.2.2'
+
+gem 'activesupport', '~> 4.2.2'
+gem 'elasticsearch-model'
+gem 'activemodel'
+gem 'typhoeus'
+gem 'activerecord'
```

Comment:
I have questions about some of these dependencies.
* If you're using `activesupport` only for the `ActiveSupport::Concern` conventions/syntax, it might be simpler to implement that internally.
* I don't see direct usage of Typhoeus.
* I don't think activerecord should be a dependency. ~We certainly wouldn't want to bring it into Gravity.~ (Just realized it's already there! WHA?!) If you need it for tests, it can be added in the .gemspec as a development dependency (see comment below). And the callbacks that I think _are_ depended on come from ActiveModel I believe.